src/engine/renderer/GeometryOptimiser.cpp

```diff
@@ -0,0 +1,259 @@
+﻿/*
+===========================================================================
+
+Daemon BSD Source Code
```

Comment:
This file seems to have old code copy and pasted so it should have the GPL license.